### PR TITLE
add statistics button component

### LINKED ISSUE #528: Adjust a statistics button to match mobile ui
- Role: only manager

### Figma design
https://www.figma.com/file/pxj9Hhk5AdOxXWJP0dq4gT/Library-App-Spec?node-id=4883%3A14678&t=whKuU6Xv4jj3INi3-0

### PR DESCRIPTION
**Pull request by vyacheslav1618**
closes #528 
Снимок экрана от 2022-12-05 22-04-25
Снимок экрана от 2022-12-05 22-05-54

There is no required SVG image in project library.

### CODE REVIEW

**Comment by vyacheslav1618** [comment]
Снимок экрана от 2022-12-05 22-38-47
Снимок экрана от 2022-12-05 22-38-35

File: packages/frontapp/src/app/components/SearchWrapper/index.tsx [hunk 1]
```diff
@@ -46,15 +47,18 @@ const SearchWrapper: FC<IProps> = ({ showSidebar }) => {
...
         <SearchByUserName />
       ) : (
-        <SearchByBookOrAuthor
-          path={
-            userRole === RolesTypes.READER
-              ? `${RoutesTypes.SEARCH}_by_name_or_author`
-              : RoutesTypes.BOOKS_STUFF
-          }
-        />
+        <>
```
Comment (Chichmarin17) [inline packages/frontapp/src/app/components/SearchWrapper/index.tsx:50]: why do we need that extra fragment wrapper?

File: packages/frontapp/src/app/components/SearchWrapper/index.tsx [hunk 2]
```diff
@@ -46,15 +47,18 @@ const SearchWrapper: FC<IProps> = ({ showSidebar }) => {
...
+            path={
+              userRole === RolesTypes.READER
+                ? `${RoutesTypes.SEARCH}_by_name_or_author`
+                : RoutesTypes.BOOKS_STUFF
+            }
+          />
+        </>
       )}
       <FiltersButton />
+      <StatisticsButton />
```
Comment (Chichmarin17) [inline packages/frontapp/src/app/components/SearchWrapper/index.tsx:61]: We should not show the statistics button for readers, render it only for managers

**Review by Chichmarin17 — CHANGES_REQUESTED** [review]

File: packages/frontapp/src/app/components/SearchWrapper/index.tsx [hunk 3]
```diff
@@ -55,6 +56,7 @@ const SearchWrapper: FC<IProps> = ({ showSidebar }) => {
         />
       )}
       <FiltersButton />
+      {isReadersPage ? null : <StatisticsButton />}
```
Comment (Chichmarin17) [inline packages/frontapp/src/app/components/SearchWrapper/index.tsx:59]: You must check not `readers` page but a `manager` role

**Review by Chichmarin17 — CHANGES_REQUESTED** [review]

**Review by ngame2011 — APPROVED** [review]

**Pull request merged**